t/lib/t/MusicBrainz/Server/Controller/WS/2/DefaultFormat.pm

```diff
@@ -0,0 +1,91 @@
+package t::MusicBrainz::Server::Controller::WS::2::DefaultFormat;
```

Comment:
I was unaware of that 1-to-1 mapping; I'll move it there and remove the MBS-9528 label.